src/main/java/io/antmedia/rest/StreamsSourceRestService.java

```diff
@@ -440,6 +437,66 @@ public Result moveRight(@ApiParam(value = "the id of the IP Camera", required =
 		return new Result(result);
 	}
 
+	@ApiOperation(value = "Start mp4 muxing", notes = "", response = Result.class)
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/mp4muxing/start/{streamId}")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Result startMp4Muxing(@ApiParam(value = "id of the stream", required = true) @QueryParam("streamId") String streamId) {
```

Comment:
There is [enableMp4Muxing](https://github.com/ant-media/Ant-Media-Server/blob/master/src/main/java/io/antmedia/rest/BroadcastRestService.java#L1331) method in BroadcastRestService.

Developers expect enableMp4Muxing to work on runtime as well and for the simplicity of the API. Remove this method and add this functionality to the enableMp4Muxing method.  

enableMp4Muxing method should enable/disable mp4 muxing both on streaming or not streaming. 
If it's streaming, it starts muxing(recording)
if it's not streaming, it records database to save the mp4 as it does right now